wiki/Tournaments/CH/CHM_1/en.md

```diff
@@ -0,0 +1,823 @@
+4. Players **are not allowed** to join in (or register for) more than one Qualifier lobby.
+5. Based on their performance in the Qualifier, players will be ranked based on their average rank combined from each individual Qualifier beatmaps.
+   - 24 players from the Latinoamerica Norte division, 24 players from the Latinoamerica Sur division, and 12 players from the España division with **the highest average rank combined from each individual Qualifier beatmaps in each division** will advance to their respective division's Swiss round.
+   - If there are two (or more) players who share the same average rank, the player that holds the higher total score combined from all ten Qualifier beatmaps will be placed in the higher seed.
+6. Failure to attend in any of the predetermined Qualifier lobbies will result in an instant elimination from the tournament.
+
+#### Swiss round rules
+
+1. All players will be matched against five other players in their respective division over the course of three weeks based on their Qualifier seed.
+2. **There are no Tiebreakers in the Swiss round.**
+   - All matches in the Swiss round are going to be played until either player obtains 5 points *or* the score gets to 4-4 (in which the match will then be considered as a *draw*).
+3. Based on their performance in the Swiss round, players will be ordered based on the following criteria:
+   - Number of wins.
+   - Number of draws.
+   - Head-to-head record against other players.
+   - Whether there are players who are disqualified from the tournament.
+4. The top 7 players in the Latinoamerica Norte division, top 7 players in the Latinoamerica Sur division, and top 2 players in the España division at the end of the Swiss round will advance to the Round of 16.
+
+#### Knock-out stage rules
+
+1. All players will be matched against each other based on their finishing position in the Swiss round.
+2. Players will compete against each other using the modified Double Elimination system.
+3. The modified Double Elimination System works as follows:
+   - Players who lose in the upper bracket can still play again on the lower bracket.
+   - Players who lose in the lower bracket will be eliminated from the tournament.
+4. **There will be no bracket reset in the Grand Final.** Instead of a traditional bracket reset system, the Grand Final will instead be played across three sets of matches in which each player needs to win at least 2 out of the 3 contested sets in order to clinch the championship title.
+5. Players who can compete in the next round are determined by the following criteria:
+   - In the Round of 16 and the Quarterfinals, each player needs to win 6 points in order to win a match. (Best-of-11)
+   - In the Semifinals onwards, each player needs to win 7 points in order to to win a match. (Best-of-13) 
+   - Players who win By default.
+   - If there are players who are disqualified from the tournament.
```

Comment:
umm i'm quite sure that this exact wording is something that i took a long time ago from a different tournament's wiki page tho (from which i've been using and copying this across many tournament articles lol)

either way yeah i changed this according to your suggestion

Reply:
we probably will need standard templates for tournament rules, because otherwise small fixes will keep happening iteratively every new tournament, which looks a bit silly